starters/nextjs-starter-approuter-ts/lib/utils.ts

```diff
@@ -15,46 +16,46 @@ export function formatDate(input: string | number): string {
   });
 }
 
-interface DateInputObject {
-  msSinceEpoch: string;
-}
-
-function isDateInputObject(v: DateInputObject | unknown): v is DateInputObject {
-  return (v as DateInputObject).msSinceEpoch != null;
-}
-
-export function getSeoMetadata(article: ArticleWithoutContent) {
-  const tags = article.tags && article.tags.length > 0 ? article.tags : [];
-  let authors = [];
-  let publishedTime = null;
-
-  // Collecting data from metadata fields
-  Object.entries(article.metadata || {}).forEach(([key, val]) => {
-    if (key.toLowerCase().trim() === "author" && val) authors = [val];
-    else if (key.toLowerCase().trim() === "date" && isDateInputObject(val))
-      publishedTime = new Date(val.msSinceEpoch).toISOString();
-  });
-
+export function getSeoMetadata(article: ArticleWithoutContent): Metadata {
+  const tags: string[] =
+    article.tags && article.tags.length > 0 ? article.tags : [];
   const imageProperties = [
+    article.metadata?.image,
     article.metadata?.["Hero Image"],
     // Extend as needed
   ]
     .filter((url): url is string => typeof url === "string")
     .map((url) => ({ url }));
+  const description = article.metadata?.description
+    ? String(article.metadata?.description)
```

Comment:
I don't have enough context yet to review the PR. But on a casual glance, I noticed this. 
optional chaining again in String(article.metadata?.description) might not necessary. I see this in other places too

Reply:
Ah yeah that's true, I'll pay more attention to these. It'd be good to catch these in linting too, I'll see if I can get a PR out adding an ESLint rule for this.

Reply:
that would be nice .. 👍